### PR TITLE
Allow to set custom image for VPN

### PR DESCRIPTION
**Pull request by jnummelin**
Allows easier experimentation with VPN

### CODE REVIEW

**Comment by SpComb** [comment]
It would be great if we could replace `kontena vpn create` with a stack file... it would need some new custom resolvers, like

```yaml
variables:
  image:
    default: kontena/openvpn:ethwe
  node:
    from:
      - host_node:
          public_ip: true
services:
  vpn:
    image: $image
    affinity:
      - node==$node
    environment:
      - OVPN_SERVER_URL=udp://{{node.public_ip}:1194
```

**Comment by SpComb** [comment]
Nothing against this PR specifically, it just feels slightly meh in general to start adding a million `kontena vpn create` / `kontena registry create` flags when we could be extending and dogfooding our stacks instead.

File: cli/lib/kontena/cli/vpn/create_command.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ class CreateCommand < Kontena::Command
 
     option '--node', 'NODE', 'Node name where VPN is deployed'
     option '--ip', 'IP', 'Node ip-address to use in VPN service configuration'
+    option '--image', 'IMAGE', 'Custom Docker image to use', hidden: true
```
Comment (SpComb) [inline cli/lib/kontena/cli/vpn/create_command.rb]: default: 'kontena/openvpn:ethwe'

**Comment by jnummelin** [comment]
I do agree that creating custom stack files is much better to try out things like this. Closing.

**Pull request closed without merging**